packages/stark-ui/src/modules/app-sidebar/components/app-sidebar.component.html

```diff
@@ -0,0 +1,20 @@
+<mat-sidenav-container #appSidenavContainer>
+	<mat-sidenav #appSidenavLeft class="stark-app-sidenav-left" [ngClass]="{'stark-app-sidenav-menu': sidenavLeftMode==='menu'}" closed mode="over" [fixedInViewport]="true"
+	[fixedBottomGap]="0">
+		<ng-container *ngIf="sidenavLeftMode==='regular'">
+			<ng-content class="regular" select="[stark-app-sidenav-left]"></ng-content>
+		</ng-container>
+		<ng-container class="menu" *ngIf="sidenavLeftMode==='menu'">
+			<ng-content select="[stark-app-sidenav-menu]"></ng-content>
+		</ng-container>
+	</mat-sidenav>
+	<mat-sidenav #appSidenavRight class="stark-app-sidenav-right" closed mode="over" position="end" [fixedInViewport]="true"
```

Comment:
What is the attribute "closed" used for? Is it really needed here? Shouldn't we close/open the sidebar programatically instead?